global_costmap:
    update_frequency: 5
    publish_frequency: 0
    static_map: true
    rolling_window: false
